docs/docs/development/new-shield.md

```diff
@@ -34,6 +34,8 @@ own prototype or handwired keyboard, it is recommended to use your own user conf
 of this guide, replace the `app/` directory in the ZMK main repository with the `config/` directory in your
 user config repository. For example, `app/boards/shields/<keyboard_name>` should now be
 `config/boards/shields/<keyboard_name>`.
+
+If you are compiling locally, you will still compile from within `zmk/app`, and you will need to provide the `DZMK_CONFIG` parameter to `west`, e.g., `-DZMK_CONFIG=/path/to/zmk-config/config`, where `/path/to` is the absolute path to your `zmk_config` directory.
```

Comment:
This was helpful for me. I would suggest perhaps specifying that the parameter should go after the `--` in the given west build command.